planets.ts

```diff
@@ -146,11 +148,17 @@ namespace space {
         createNeptune(neptuneCenter, Radius.Neptune * scale);
     }
 
+    /**
+     * Builds the planet Mercury with gray concrete.
+     * @param center the position where the center of the planet will be
```

Comment:
Nit: Not convinced this is necessary so feel free to skip, but it might be nice to say the name of the planet instead of just "the planet" for each planet.